models/members.js

```diff
@@ -148,8 +148,31 @@ const fetchUsersWithRole = async (role) => {
   }
 }
 
+/**
+ * changes the role of a new user to member
+ * @param userId { String }: User id of user to be modified
+ * @return { Object }: whether moveToMember was successful or not and whether user is already a member or not
+ */
+
+const addArchiveRoleToMembers = async (userId) => {
+  try {
+    const userDoc = await userModel.doc(userId).get()
+    const user = userDoc.data()
+    if (user?.roles?.archivedMember) return { isArchived: true }
```

Comment:
Since we're having archivedMember: true in both the cases, what if we seperate out that as a variable so we reduce duplication?

Reply:
yes, we can do that.

Reply:
Where's the updated code?